airbrakes/mock/mock_imu.py

```diff
@@ -19,12 +19,14 @@ class MockIMU(IMU):
 
     __slots__ = ()
 
-    def __init__(self, log_file_path: Path, real_time_simulation: bool):
+    def __init__(self, log_file_path: Path, real_time_simulation: bool, start_after_log_buffer: bool = False):
         """
         Initializes the object that pretends to be an IMU for testing purposes by reading from a log file.
         :param log_file_name: The name of the log file to read data from.
         :param real_time_simulation: Whether to simulate a real flight by sleeping for a set period, or run at full
             speed, e.g. for using it in the CI.
+        :param start_after_log_buffer: Whether to send the data packets only after the log buffer
```

Comment:
Is this so when testing you don't have to wait for it to launch?

Reply:
tbh @wlsanderson had problems with this for apogee prediction, that's why I did this. But it also helps quicken things up ever so slightly